### PR TITLE
Rewords and formats instructions, and fixes reference to run-reader.sh

### PR DESCRIPTION
**Pull request by cadmiumcat**
### What
After following the [getting started guide](https://github.com/ONSdigital/dp/blob/master/guides/GETTING_STARTED.md), I've made some changes that will hopefully make it slightly easier to follow. The changes: 
- metion Nepture migration
- add a table for prerequisites
- fix zebedee command (instructions now say run-cms.sh instead of run-reader.sh)
- change layout to make commands easier to copy
- re-arrange instructions for running CMD
- rearrange instructions for SSH mac setup

### How to review

Check that the instructions still make sense 😆. I'm particularly shaky on the bit about [running CMS](https://github.com/ONSdigital/dp/blob/feature/getting-started/guides/INSTALLING.md#cmd)

### Who can review

Any dev

### CODE REVIEW

File: guides/INSTALLING.md [hunk 1]
```diff
@@ -14,146 +14,142 @@ In the below, the installation of each app is typically one of:
...
-  - `$ git clone git@github.com:ONSdigital/dp-observation-importer`
 * [dp-hierarchy-builder](https://github.com/ONSdigital/dp-hierarchy-builder)
-  - `$ git clone git@github.com:ONSdigital/dp-hierarchy-builder`
 * [dp-search-builder](https://github.com/ONSdigital/dp-search-builder)
-  - `$ git clone git@github.com:ONSdigital/dp-search-builder`
 * [dp-publishing-dataset-controller](https://github.com/ONSdigital/dp-publishing-dataset-controller)
-  - `$ git clone git@github.com:ONSdigital/dp-publishing-dataset-controller`
+
+  ```bash
+  git clone git@github.com:ONSdigital/dp-recipe-api`
```
Comment (gedge) [inline guides/INSTALLING.md:123]: trailing backticks

File: guides/INSTALLING.md [hunk 2]
```diff
@@ -182,7 +178,7 @@ When the startup files are updated, to load the new changes into your shell, eit
 
 You should put the below _env vars_ in your [startup file](#startup-file).
 
-VAR_NAME | note
+Varialbe name | note
```
Comment (gedge) [inline guides/INSTALLING.md]: typo

File: guides/INSTALLING.md [hunk 3]
```diff
@@ -14,146 +14,142 @@ In the below, the installation of each app is typically one of:
...
-
-  NB. See `dp-compose` README for configuration of Docker Desktop resources
 
+Software | Install | Notes
+-------- | ------- | ----- |
+[Java 8 JDK (OpenJDK)](https://openjdk.java.net/install/) | `$ brew cask install adoptopenjdk8`
+[Maven](https://maven.apache.org/)                        | `$ brew install maven`
+[Docker](https://www.docker.com/get-started)              | `$ brew cask install docker`
+ Docker Compose                                           | `$ brew install docker-compose`
+[Cypher Shell](https://neo4j.com/docs/operations-manual/current/tools/cypher-shell/)                                | `$ brew install cypher-shell`
```
Comment (gedge) [inline guides/INSTALLING.md:24]:
```suggestion
[Cypher Shell](https://neo4j.com/docs/operations-manual/current/tools/cypher-shell/)                                | `$ brew install cypher-shell` | deprecated (not needed if using Neptune over Neo4j)
```

File: guides/INSTALLING.md [hunk 4]
```diff
@@ -14,146 +14,142 @@ In the below, the installation of each app is typically one of:
...
+[Node.js and npm](https://nodejs.org/en/)                 | Install the LTS version with: `$ brew install node@12`  | Append `export PATH="/usr/local/opt/node@12/bin:$PATH"` to the your `.zshrc` or `.basrc` file and restart your terminal.
```
Comment (gedge) [inline guides/INSTALLING.md:25]:
```suggestion
[Node.js and npm](https://nodejs.org/en/)                 | `$ brew install node@12` (LTS version) | Append `export PATH="/usr/local/opt/node@12/bin:$PATH"` to your startup files and restart your terminal.
```

File: guides/INSTALLING.md [hunk 5]
```diff
@@ -182,7 +178,7 @@ When the startup files are updated, to load the new changes into your shell, eit
 
 You should put the below _env vars_ in your [startup file](#startup-file).
 
-VAR_NAME | note
+Varialbe name | note
 --- | ---
```
Comment (HeJones) [inline guides/INSTALLING.md:182]: Just a small typo

File: guides/INSTALLING.md [hunk 6]
```diff
@@ -14,146 +14,142 @@ In the below, the installation of each app is typically one of:
...
 
 ---
 
 ## Clone the services
 
-Clone the GitHub repos for [web](#web-journey), [publishing](#publishing-journey) and/or [CMD](#cmd-journeys).
+Clone the GitHub repos for [web](#web-journey), [publishing](#publishing-journey) and/or [CMD](#cmd-journeys) (Customise My Data).
 
-- [Web](#web-journey) - These apps make up the public facing website providing read only access to published content, and will be enough strictly to work on website content types other than filterable datasets (e.g. bulletins, articles, timeseries, datasets). 
+- [Web](#web-journey) - These apps make up the public facing website providing **read only access** to published content, and will be enough strictly to work on website content types other than filterable datasets (e.g. bulletins, articles, timeseries, datasets).
```
Comment (gedge) [inline guides/INSTALLING.md:52]:
```suggestion
- [Web](#web-journey) - These apps make up the public-facing website providing **read-only access** to published content, and will be enough strictly to work on website content types other than filterable datasets (e.g. bulletins, articles, timeseries, datasets).
```

File: guides/INSTALLING.md [hunk 7]
```diff
@@ -14,146 +14,142 @@ In the below, the installation of each app is typically one of:
...
 
 - [Publishing](#publishing-journey) - The "publishing journey" gives you all the features of web together with an internal interface to update, preview and publish content. All content is encrypted and requires authentication.
 
-- [CMD](#cmd-journeys) - apps will support the filterable dataset journey, and would mean you have every possible service running.
+- [CMD](#cmd-journeys) - apps will support the filterable dataset journey, and would mean you have every possible service running.
```
Comment (gedge) [inline guides/INSTALLING.md:56]: trailing whitespace

File: guides/INSTALLING.md [hunk 8]
```diff
@@ -14,146 +14,142 @@ In the below, the installation of each app is typically one of:
...
-  - `$ git clone git@github.com:ONSdigital/dp-filter-api`
 * [dp-dataset-exporter](https://github.com/ONSdigital/dp-dataset-exporter)
-  - `$ git clone git@github.com:ONSdigital/dp-dataset-exporter`
 * [dp-dataset-exporter-xlsx](https://github.com/ONSdigital/dp-dataset-exporter-xlsx)
-  - `$ git clone git@github.com:ONSdigital/dp-dataset-exporter-xlsx`
 * [dp-download-service](https://github.com/ONSdigital/dp-download-service)
-  - `$ git clone git@github.com:ONSdigital/dp-download-service`
+
+  ```bash
+  git clone git@github.com:ONSdigital/dp-search-api`
```
Comment (gedge) [inline guides/INSTALLING.md:145]: trailing backticks

File: guides/INSTALLING.md (hunk 2 above)
Comment (gedge) [inline guides/INSTALLING.md]:
```suggestion
Variable name | note
```

File: guides/INSTALLING.md [hunk 9]
```diff
@@ -195,57 +191,51 @@ VAR_NAME | note
...
+```
 
 Return to the [Getting Started](https://github.com/ONSdigital/dp/blob/master/guides/GETTING_STARTED.md) guide for next steps.
 
 --------------
 
 ## Running the apps
 
-Before running web or publishing, please make sure to run [dp-compose](https://github.com/ONSdigital/dp-compose) using the `./run.sh` command (in the dp-compose repo) to run the supporting services.
+Run [dp-compose](https://github.com/ONSdigital/dp-compose) using the `./run.sh` command (in the dp-compose repo) to run the supporting services. As well as `Vault`, e.g. `$ vault server -dev`.
```
Comment (gedge) [inline guides/INSTALLING.md:218]:
```suggestion
Run [dp-compose](https://github.com/ONSdigital/dp-compose) using the `$ ./run.sh` command (in the dp-compose repo) to run the supporting services. As well as Vault, e.g. `$ vault server -dev`.
```

File: guides/INSTALLING.md [hunk 10]
```diff
@@ -195,57 +191,51 @@ VAR_NAME | note
...
 
 Most applications can be run using the `$ make debug` command, but deviations are all documented below:
 
-#### Web
+### Web
+Run all the services in the [web journey](#web-journey)
```
Comment (gedge) [inline guides/INSTALLING.md:223]: Please use blank lines between headers and body lines.

**Comment by HeJones** [comment]
It looks good to me, but it might be worth having someone else look at it too, particularly the running CMS section

File: guides/INSTALLING.md [hunk 11]
```diff
@@ -258,35 +248,38 @@ and also run the following:
 
 Florence will be available at http://localhost:8081/florence/login
 
-The website will be available at http://localhost:8081 (only available after a successful login into Florence - see the README.md in the florence directory)
+The website will be available at http://localhost:8081 after a successful login into florence. Login details are in the [florence repository](https://github.com/ONSdigital/florence/blob/develop/USAGE.md).
```
Comment (gedge) [inline guides/INSTALLING.md:251]:
```suggestion
The website will be available at [http://localhost:8081](http://localhost:8081) after a successful login into florence. Login details are in the [florence repository](https://github.com/ONSdigital/florence/blob/develop/USAGE.md).
```

File: guides/INSTALLING.md (hunk 5 above)
Comment (cadmiumcat) [reply guides/INSTALLING.md:182]: varialalbel*

File: guides/INSTALLING.md [hunk 12]
```diff
@@ -14,146 +14,142 @@ In the below, the installation of each app is typically one of:
...
+[Docker](https://www.docker.com/get-started)              | `$ brew cask install docker`
+ Docker Compose                                           | `$ brew install docker-compose`
+[Cypher Shell](https://neo4j.com/docs/operations-manual/current/tools/cypher-shell/)                                | `$ brew install cypher-shell`
+[Node.js and npm](https://nodejs.org/en/)                 | Install the LTS version with: `$ brew install node@12`  | Append `export PATH="/usr/local/opt/node@12/bin:$PATH"` to the your `.zshrc` or `.basrc` file and restart your terminal.
+[GoConvey](https://github.com/smartystreets/goconvey#installation)  |
+[GhostScript](https://www.ghostscript.com/download.html)  |   `$ brew install ghostscript`                          | Required for [Babbage](https://github.com/onsdigital/babbage)
+[Vault](https://www.vaultproject.io/intro/getting-started/install.html) | `$ brew install hashicorp/tap/vault`      | Required for running Florence.
+[jq](https://stedolan.github.io/jq/)                      | `$ brew install jq`                                     | A handy JSON tool (for debugging website content)
```
Comment (gedge) [inline guides/INSTALLING.md:29]:
```suggestion
[jq](https://stedolan.github.io/jq/)                      | `$ brew install jq`                                     | A handy JSON tool (for debugging website content and much more)
```

**Review by gedge — CHANGES_REQUESTED** [review]
lots of good stuff, just a few tweaks

**Review by gedge — APPROVED** [review]
LGTM - thanks for the improvements 👍

**Pull request merged**